src/content/test/text-legibility/reflow.tsx

```diff
@@ -27,8 +27,8 @@ export const infoAndExamples = create(({ Markup }) => (
             <li>
                 Better: Do not require users to scroll in the direction of reading:
                 <ul>
-                    <li>the script is read horizontally (left-to-right or right-to-left), text should scroll vertically.</li>
-                    <li>the script is read vertically (top-to-bottom), text should scroll horizontally.</li>
+                    <li>if the script is read horizontally (left-to-right or right-to-left), text should scroll vertically.</li>
```

Comment:
I think using sentence case (`if` -> `If`) would be more consistent with the other content